components/molecule/tabs/src/styles/index.scss

```diff
@@ -3,7 +3,13 @@ $base-class-full-width: '#{$base-class}--fullWidth';
 $item-base-class: '#{$base-class}-item';
 
 #{$base-class} {
+  background-color: $bgc-sui-molecule-tabs;
   border-radius: $bdrs-sui-molecule-tabs;
+  overflow: hidden;
```

Comment:
this is needed in order to avoid quirks when the content bg color is different to the component bg color.